Plots/Contours/NCL_conwomap_5.py

```diff
@@ -0,0 +1,126 @@
+"""
+NCL_conwomap_5.py
+=================
+This script illustrates the following concepts:
+   - Drawing a simple contour plot
+   - Making an axis logarithmic in a contour plot
+   - Changing the labels and tickmarks on a contour plot
+   - Creating a main title
+   - Using the geocat-comp method interp_hybrid_to_pressure
```

Comment:
Would be great to provide an embedded link to documentation of this function here. The link should be:

https://geocat-comp.readthedocs.io/en/latest/user_api/generated/geocat.comp.interp_hybrid_to_pressure.html#geocat.comp.interp_hybrid_to_pressure